bms-language.js

```diff
@@ -0,0 +1,30 @@
+const LanuageMessage = {
+    en:{
+        message:[
+                'Breach : too low',
+                'Warning : approaching low Breach',
+                'OK: Normal',
+                'Warning : approaching high Breach',
+                'Breach : too high',
+            ],
+        parameterName: {temperature:"Temperature", soc:"State of Charge", chargeRate: "Charge Rate"}
+    },
+    ger:{
+        message:[
+            'Bruch: zu niedrig',
+            'Warnung: Tiefer Durchbruch naht',
+            'OK: Normal',
+            'Warnung: Hoher Durchbruch steht bevor',
+            'Verstoß: zu hoch'
+        ],
+        parameterName: {temperature:'Temperatur', soc:'Ladezustand', chargeRate:"Ladestrom"}
+    },
+}
+function messageCodeFromLevelOfBreach(levelOfBreach)  {
+    return levelOfBreach + 2
```

Comment:
The `+ 2` here is not understandable without reading the `bms-checkParameter.js`
Someone changing this file may not see the other one, or someone increasing the number of 'levels' in `bms-checkParameter.js` may not see this file. So writing comments in either file will not guarantee mistake-proofing of modifications.
Can you think of ways to mistake-proof it, without adding comments?
Hint: think of tests to enforce the sync between the levels and the messages

Reply:
I created a separate file MessageFromLevelOfBreach.js that maps `levelOfBreach`  and `LanguageMessages`  and returns the message